{"uuid":"112eaf9a-c680-4239-9dc0-0fbe94b09538","historyId":"999b1154b123f2a28d0fff10a1bb06bd","fullName":"testPackage.Test01.portalLogin","labels":[{"name":"package","value":"testPackage.Test01"},{"name":"testClass","value":"testPackage.Test01"},{"name":"testMethod","value":"portalLogin"},{"name":"parentSuite","value":"Surefire suite"},{"name":"suite","value":"Surefire test"},{"name":"subSuite","value":"testPackage.Test01"},{"name":"host","value":"users-MacBook-Pro.local"},{"name":"thread","value":"8333@users-MacBook-Pro.local.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"story","value":"Valid Login"},{"name":"epic","value":"Suite"},{"name":"feature","value":"Login Feature"}],"links":[],"name":"portalLogin","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"java.io.FileNotFoundException: /Users/user/Documents/Selenium Project/login_credentials.csv (No such file or directory)","trace":"java.lang.RuntimeException: java.io.FileNotFoundException: /Users/user/Documents/Selenium Project/login_credentials.csv (No such file or directory)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:53)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:154)\n\tat org.testng.internal.Parameters.handleParameters(Parameters.java:767)\n\tat org.testng.internal.Parameters.handleParameters(Parameters.java:704)\n\tat org.testng.internal.invokers.ParameterHandler.handleParameters(ParameterHandler.java:70)\n\tat org.testng.internal.invokers.ParameterHandler.createParameters(ParameterHandler.java:49)\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:882)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:192)\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1596)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:808)\n\tat org.testng.TestRunner.run(TestRunner.java:603)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:429)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:326)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1169)\n\tat org.testng.TestNG.runSuites(TestNG.java:1092)\n\tat org.testng.TestNG.run(TestNG.java:1060)\n\tat org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:155)\n\tat org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:169)\n\tat org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:88)\n\tat org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:137)\n\tat org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)\n\tat org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)\n\tat org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)\n\tat org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)\nCaused by: java.io.FileNotFoundException: /Users/user/Documents/Selenium Project/login_credentials.csv (No such file or directory)\n\tat java.base/java.io.FileInputStream.open0(Native Method)\n\tat java.base/java.io.FileInputStream.open(FileInputStream.java:213)\n\tat java.base/java.io.FileInputStream.<init>(FileInputStream.java:152)\n\tat java.base/java.io.FileInputStream.<init>(FileInputStream.java:106)\n\tat java.base/java.io.FileReader.<init>(FileReader.java:60)\n\tat testPackage.ReadCSVHelper.readDataFromCSV(ReadCSVHelper.java:12)\n\tat testPackage.Test01.userData(Test01.java:374)\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:136)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:49)\n\t... 30 more\n"},"stage":"finished","description":"Test the login functionality with valid credentials.","steps":[],"attachments":[],"parameters":[],"start":1730945798719,"stop":1730945798719}